.papr.yml

```diff
@@ -79,14 +81,25 @@ tests:
 inherit: true
 required: true
 
-context: f26-curl-openssl
+context: f26-libsoup
 
-packages:
-  - pkgconfig(libcurl)
-  - pkgconfig(openssl)
+env:
+  CONFIGOPTS: "--without-curl --without-openssl"
+
+tests:
+  - ci/build-check.sh
+
+---
+
+inherit: true
+required: true
+
+context: f26-introspection-tests
 
 env:
-  CONFIGOPTS: "--with-curl --with-openssl"
+    CFLAGS: ''  # Reset ASAN
+    CONFIGOPTS: "--without-curl --without-openssl"
+    INSTALLED_TESTS_PATTERN: "tests/test-sizes.js"
```

Comment:
Also tests/test-corruption.sh tests/test-pull-corruption.sh tests/test-core.js and tests/test-sysroot.js?

Otherwise looks good to me - the code that @jlebon commented on has just been hoisted higher up, not modified, so whether it actually needs to be testing for G_FILE_TYPE_REGULAR seems orthogonal to this bug.